src/antsibull_docs/write_docs.py

```diff
@@ -650,6 +651,8 @@ async def write_plugin_lists(collection_name: str,
         disabled.  This will disable breadcrumbs but save on memory usage.
     :kwarg for_official_docsite: Default False.  Set to True to use wording specific for the
         official docsite on docs.ansible.com.
+    :kwarg squash_hierarchy: If set to ``True``, no directory hierarchy will be used.
+        Undefined behavior if documentation for multiple collections are created.
```

Comment:
@felixfontein Finding it a little hard to grok this line. Am I correct that `squash_hierarchy: True` has undefined behavior if you are creating docs for multiple collections?

Reply:
Yes. That's why it's impossible to let the CLI do that:
- https://github.com/ansible-community/antsibull-docs/blob/main/src/antsibull_docs/cli/antsibull_docs.py#L121
- https://github.com/ansible-community/antsibull-docs/blob/main/src/antsibull_docs/cli/antsibull_docs.py#L150